src/pool.rs

```diff
@@ -34,6 +34,24 @@ impl<T: TaskCell + Send> ThreadPool<T> {
         self.remote.spawn(t);
     }
 
+    /// Scale workers of the thread pool, the new thread count should be less
+    /// than or equal to the max_thread_count, otherwise it will become no-op
```

Comment:
It should follow the same behavior of `Builder::core_thread_count`.

Reply:
Sure,

Reply:
Fixed (Sorry, I misunderstood what you meant by the same comment last time)